### ISSUE TITLE
Don't find the latest JAR as per latest upgrade for parse 2.0.14

### ISSUE THREAD

**Issue by vykugithub** [saurfang/spark-sas7bdat#76]
Don't see latest JAR files here for the latest commits which is parse jar upgrade 2.0.14
https://github.com/saurfang/spark-sas7bdat/releases

**Comment by dcephilpott** [comment]
Yes - I think this is causing some dependency issues, could we have a new JAR built so we can use Parso 2.0.14. Thanks

**Comment by janosek** [comment]
I second this request.

**Comment by leandrohmvieira-db** [comment]
I just had the same error, is there any workaround on this yet? updating dependencies from Maven and do a clean build fix it?
